### DISCUSSION TITLE
When will the paginated datatable be available?

### DISCUSSION THREAD

**Discussion by charliethacker** [flet-dev/flet#1847] [Q&A]
### Question

Flutter with Dart has a paginated data table.    Will this component be available on Flet?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
I made one using the already available Flet controls: https://github.com/ndonkoHenri/Flet-Custom-Controls/tree/master/Paginated%20DataTable

Let me know how you find it...
